.github/workflows/build-release.yml

```diff
@@ -3,6 +3,12 @@ name: "[builder] CI for releases"
 on:
   repository_dispatch:
     types: build
+  release:
+    types: [published]
```

Comment:
Please note that this event will only be triggered if there's a new release in https://github.com/HenryQW/Awesome-TTRSS/releases

Reply:
I know about release events, but the project has never used them. It's better to use nightly builds as the testing branch for development, and tags and release events for final user releases. I've encountered many instances where nightly builds failed to start properly in the past.